fixtures/workers-with-assets/tests/index.test.ts

```diff
@@ -3,211 +3,221 @@ import { fetch } from "undici";
-		let response = await fetch(`http://${ip}:${port}/index.html`);
-		let text = await response.text();
-		expect(response.status).toBe(200);
-		expect(text).toContain(`<h1>Hello Workers + Assets World 🚀!</h1>`);
-
-		response = await fetch(`http://${ip}:${port}/about/index.html`);
-		text = await response.text();
-		expect(response.status).toBe(200);
-		expect(text).toContain(`<p>Learn more about Workers with Assets soon!</p>`);
-	});
-
-	it("should fallback to the user Worker if there are no assets at a given path ", async ({
-		expect,
-	}) => {
-		// Requests should hit the Asset Worker *first*, then try the user Worker
-		const response = await fetch(`http://${ip}:${port}/no-assets-here`);
-		const text = await response.text();
-		expect(text).toContain(
-			"There were no assets at this route! Hello from the user Worker instead!"
-		);
-	});
-
-	// html_handling defaults to 'auto-trailing-slash'
-	it("should `/` resolve to `/index.html` ", async ({ expect }) => {
-		const response = await fetch(`http://${ip}:${port}/`);
-		const text = await response.text();
-		expect(text).toContain("<h1>Hello Workers + Assets World 🚀!</h1>");
-	});
-
-	it("should handle content types correctly on asset routes", async ({
-		expect,
-	}) => {
-		let response = await fetch(`http://${ip}:${port}/index.html`);
-		let text = await response.text();
-		expect(response.status).toBe(200);
-		expect(response.headers.get("Content-Type")).toBe(
-			"text/html; charset=utf-8"
-		);
-
-		response = await fetch(`http://${ip}:${port}/README.md`);
-		text = await response.text();
-		expect(response.status).toBe(200);
-		expect(response.headers.get("Content-Type")).toBe(
-			"text/markdown; charset=utf-8"
-		);
-		expect(text).toContain(`Welcome to Workers + Assets YAY!`);
-
-		response = await fetch(`http://${ip}:${port}/yay.txt`);
-		text = await response.text();
-		expect(response.status).toBe(200);
-		expect(response.headers.get("Content-Type")).toBe(
-			"text/plain; charset=utf-8"
-		);
-		expect(text).toContain(`.----------------.`);
-
-		response = await fetch(`http://${ip}:${port}/lava-lamps.jpg`);
-		expect(response.status).toBe(200);
-		expect(response.headers.get("Content-Type")).toBe("image/jpeg");
-
-		response = await fetch(`http://${ip}:${port}/totallyinvalidextension.greg`);
-		expect(response.status).toBe(200);
-		expect(response.headers.has("Content-Type")).toBeFalsy();
-	});
-
-	it("should return 405 for non-GET or HEAD requests on routes where assets exist", async ({
-		expect,
-	}) => {
-		// these should return the error and NOT be forwarded onto the user Worker
-		// POST etc. request -> RW -> AW -> check manifest --405--> RW --405--> eyeball
-
-		// methods as per https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods
-		// excl. TRACE and CONNECT which are not supported
-
-		let response = await fetch(`http://${ip}:${port}/index.html`, {
-			method: "POST",
+const devCmds = [{ args: [] }, { args: ["--x-assets-rpc"] }];
+
+describe.each(devCmds)(
```

Comment:
no tests changes were added here!! We are basically running the existing set of tests for both `wrangler dev` and `wrangler dev --x-assets-rpc`